__init__.py

```diff
@@ -350,17 +356,22 @@ def calculate_anharmonic_thermo(self):
                 Z_mode = 1./(1.-np.exp(-e.real/(self.kT)))
                 ZPE = 0.5 * e.real
                 e_min_exitation = e.real
+                x = e_min_exitation/self.kT
+                self.entropic_energy_modes.append(self.kT*(x/(np.exp(x)-1.) - np.log(1.-np.exp(-x))))
+
             else:
                 ZPE = 0.
                 Z_mode = 1.
                 e_min_exitation = 0.
+                self.entropic_energy_modes.append(0.)
 
+            x = 0.
```

Comment:
I copied this from the HarmonicThermo module of ASE. I know x doesn't mean anything; any suggestion how to call E/kT?